app/router.js

```diff
@@ -7,8 +7,7 @@ const Router = Ember.Router.extend({
 });
 
 Router.map(function() {
-//  this.route('reminders');
-  this.route('reminders', { path: '/' });
+  this.route('reminders', { path: '/reminders' });
```

Comment:
The path on the right hand side here is redundant - if it matches the same string as 'reminders' Ember knows to go there by default. Modify this to just be `this.route('reminders');`